bouncer/commands/send_sol.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env -S pnpm tsx
+// INSTRUCTIONS
+//
+// This command takes two arguments.
+// It will fund the ethereum address provided as the first argument with the amount
```

Comment:
This comment still references Ethereum addresses and ETH

Reply:
Missed that one. Fixed!